drivers/network/dd/ne2000/CMakeLists.txt

```diff
@@ -11,4 +11,4 @@ add_pch(ne2000 include/ne2000.h SOURCE)
 set_module_type(ne2000 kernelmodedriver)
 add_importlibs(ne2000 ndis ntoskrnl hal)
 add_cd_file(TARGET ne2000 DESTINATION reactos/system32/drivers FOR all)
-add_driver_inf(ne2000 netne2000.inf)
+add_driver_inf(ne2000 netne.inf)
```

Comment:
Note:
This one could be renamed `netne2k.inf`, if ever wanted.

Reply:
Yes, that would be possible as well. I do agree.

Reply:
Yeah, Novell NE1000 exists, though it's an old ISA card.